apps/web/src/app/api/documentLogs/generate-csv/route.ts

```diff
@@ -0,0 +1,39 @@
+import { Workspace } from '@latitude-data/core/browser'
+import { BadRequestError } from '@latitude-data/core/lib/errors'
+import { generateCsvFromDocumentLogs } from '@latitude-data/core/services/documentLogs/generateCsvFromDocumentLogs'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { NextRequest, NextResponse } from 'next/server'
+
+export const GET = errorHandler(
```

Comment:
Wait. This should be a server action via POST

Reply:
Hmmm, idk. It's just a way to fetch the data in a different way, but it does not perform any action whatoever.

Reply:
Ah you're right just generate not save +1